test/common_utils.py

```diff
@@ -16,6 +16,17 @@
 
 
 def get_bool_env_var(name, *, exist_ok=False, default=False):
+    """Gets the value of a boolean environment variable, by mapping common bool-ish
+    values to a :class:`bool`.
+
+    Args:
+        name: Name of the environment variable.
+        exist_ok: Returns ``True`` if the environment variable exists regardless of the
+            value it holds. This is useful for checking if an environment variable
+            exists even if it holds no value.
+        default: Value to return if the environment variable doesn't exist. Defaults to
+            ``False``.
```

Comment:
the default parameter doesn't seem to be used (same for `run_on_env_var`). Could we remove it to simplify the logic?